.gitpod.yml

```diff
@@ -1,35 +1,56 @@
 # Learn more about this file at https://www.gitpod.io/docs/references/gitpod-yml
 tasks:
   - name: Service Server
-    # Get Web API urls for use in Next build
+    openMode: split-left
+    # Copying env.development and replacing 'localhost' wit Gitpod urls
+    # Instead of rebuilding the nginx-base frontend when environment variables change
+    # We build the frontend directly through 'npm run dev' to use the latest .env file
+    # All services are listed here under src/api/readme.md'
     before: |
-      eval $(gp env -e GITPOD_API_URL=$(gp url 3000))
-      eval $(gp env -e GITPOD_WEB_URL=$(gp url 8000))
-      eval $(gp env -e GITPOD_MICROSERVICE_URL=$(gp url 8443))
+      eval $(gp env -e API_URL=$(gp url 3000))
+      eval $(gp env -e WEB_URL=$(gp url 8000))
+      eval $(gp env -e API_HOST=$(gp url 8443))
+      eval $(gp env -e SERVICES="status image auth search posts feed-discovery users parser planet redis elasticsearch login firebase")
```

Comment:
If we want to try and trim this down a bit we could  remove `planet`. It's not needed in development (unless you're specifically working with it), which almost no one will be.

Reply:
I trimmed planet.